cli/src/main.rs

```diff
@@ -0,0 +1,261 @@
+//! A collection of tools, mostly related to importing maps and scenarios. These are bundled as a
+//! single executable to reduce the cost of static linking in the release's file size.
```

Comment:
ya know, I didn't really think about the binary size implications. But that makes sense. 👍